.github/workflows/go.yml

```diff
@@ -24,8 +24,8 @@ jobs:
             fail-fast: false
             matrix:
                 go:
-                    - '1.18'
```

Comment:
I believe we briefly discussed this awhile ago in a meeting about the Go API design. We discussed Go 1.18 as a minimum version because it added support for generics. At the moment, its not yet clear whether we'll end up needing generics or not.